components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -99,7 +99,7 @@ class CrashReporter(
     /**
      * Get a copy of the crashBreadCrumbs
      */
-    internal fun crashBreadcrumbsCopy(): ArrayList<Breadcrumb> {
+    fun crashBreadcrumbsCopy(): ArrayList<Breadcrumb> {
```

Comment:
`@VisibleForTests`, perhaps? IIUC you made this public to access it from the tests in lib-crash-sentry, right?

Reply:
👍🏽